### PR TITLE
contribs.py: fixed the case of empty string

### PR DESCRIPTION
**Pull request by emmanuelle**
I tried to execute ``contribs.py`` and it was crashing because of an empty string. This PR is a quick fix for empty strings.

### CODE REVIEW

**Comment by blink1073** [comment]
Maybe execute this file in the travis script?

**Comment by emmanuelle** [comment]
@blink1073 I added the execution of ``contribs.py`` in ``travis_script.sh``. Is it OK for you? Should we redirect the output to a file to avoid cluttering the standard output? 

Also, I'm sure there exists a way to obtain the tag of the latest release but for the sake of testing I think it's fine to hardcode it.

File: tools/travis_script.sh [hunk 1]
```diff
@@ -111,3 +111,7 @@ fi
 nosetests $TEST_ARGS
 
 section_end "Test.with.optional.dependencies"
+
+section "Prepare.release"
+doc/release/contribs.py 76b7e42
```
Comment (blink1073) [inline tools/travis_script.sh]: maybe try `HEAD`?

**Comment by blink1073** [comment]
Ah, the reason it can't find that commit is because Travis checks out our repo at a depth of 50, so it isn't there.

**Comment by blink1073** [comment]
So, you could use something like `HEAD~10`

**Comment by blink1073** [comment]
Merry Christmas, by the way, @emmanuelle!

**Comment by emmanuelle** [comment]
Thanks a lot for the detective work @blink1073 ! And merry Christmas to you too!

**Comment by blink1073** [comment]
`subprocess.check_output` is not available on Python 2.6, you'll have to wrap this in a Python version check.

**Comment by stefanv** [comment]
:palm_tree: <-- Closest thing I could find to a Christmas tree.  Merry Christmas :)

**Comment by blink1073** [comment]
:evergreen_tree: <-- you mean this one, @stefanv?  Merry Christmas!

**Comment by stefanv** [comment]
What.  Now I also learnt something about the way GitHub auto-completed these icons.  Typing ":e" does not show me the :evergreen_tree: !

**Comment by stefanv** [comment]
:christmas_tree: would also work, I guess.

**Comment by blink1073** [comment]
touché

**Comment by emmanuelle** [comment]
I'll rebase -i once Travis is happy, the commit stack is really ugly :-).

**Comment by stefanv** [comment]
It's happy! :fireworks:

**Comment by emmanuelle** [comment]
LGTM, but I'll respect the protocol and won't merge my own PR :-). Anybody for the green button?

**Comment by soupault** [comment]
@emmanuelle would you mind adding the reminder on 2.6 hack to `TODO.txt`?

**Comment by emmanuelle** [comment]
@soupault done!

**Comment by blink1073** [comment]
Merging, since this file is not tested.  Thanks @emmanuelle!

**Pull request merged**